### PR TITLE
Adding new argument and functionality for running in 3 containers to …

### PR DESCRIPTION
**Pull request by cjames23**
Adding the ability to run within 3 different containers which would be closer to how MWAA does the runtime. 

*Issue #, if available:*

*Description of changes:*
Added a new argument called start-celery for starting three containers and using the celery executor. 

Added a new docker compose file that adds three containers 1 each for the web server, scheduler and worker. 

Added required args to the entrypoint to allow having the three containers. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: mwaa-local-env [hunk 1]
```diff
@@ -76,6 +77,9 @@ reset-db)
 start)
    docker-compose -f ./docker/docker-compose-local.yml up
    ;;
+start-celery)
+   docker compose -f ./docker/docker-compose-multi.yml up
```
Comment (o-nikolas) [inline mwaa-local-env:81]: I think we should either stick with `celery` or `multi` as the nomenclature for this setup. Right now both names are used in various places.

**Review by o-nikolas — CHANGES_REQUESTED** [review]

**Review by o-nikolas — CHANGES_REQUESTED** [review]
You'll also need to update the README in various places to outline what this setup is and how to use it. The "What this repo contains" section will have to be updated as well.

File: mwaa-local-env (hunk 1 above)
Comment (cjames23) [reply mwaa-local-env:81]: I do not have a preference here so I can change it to be multi as the nomenclature.

**Comment by cjames23** [comment]
@o-nikolas I have the readme changes in the works, I will work on getting those pushed as well as changing to use multi for the naming.

**Comment by john-jac** [comment]
This is a major refactor that is beyond the scope of local-runner at this time. Such a refactor may be considered in future releases.

**Pull request closed without merging**